Database-Features/How-to-search-Exasols-audit-tables-to-see-which-users-had-the-most-SQL-queries-in-a-given-period.md

```diff
@@ -7,35 +7,41 @@ We have an unusual peak in our DQL queries and would like to know which user wit
 
 ![Peak](images/PeggySchmidtMittenzweiDQLHighPeak2.png)
 
-This diagram visualizes the table [EXA_SQL_HOURLY](https://docs.exasol.com/db/latest/sql_references/system_tables/statistical/exa_dba_audit_sql.htm) for the attribute COMMAND_CLASS with the value DQL.
+This diagram visualizes one specific metric for COMMAND_CLASS = 'DQL' from the table table [EXA_SQL_HOURLY](https://docs.exasol.com/db/latest/sql_references/system_tables/statistical/exa_dba_audit_sql.htm).
```

Comment:
Please don't take all comments as literal written in stone. I actually wanted to know WHICH metric? (Without having to look at the diagram title)